### ISSUE TITLE
Feature Request: add SSL / TLS support

### ISSUE THREAD

**Issue by bakirov** [surgemq/surgemq#8]
Hello!

Feature request for a future: do you consider adding SSL / TLS support, so we can have MQTT in SurgeMQ to work over SSL / TLS?

Thanks!

**Comment by zhenjl** [comment]
Yes, that's the intent though it maybe a bit before I get to it. I will keep this open until then.

**Comment by n1try** [comment]
+1

**Comment by hello2dj** [comment]
+1
